iocore/eventsystem/I_Thread.h

```diff
@@ -138,18 +146,10 @@ class Thread
   ProxyAllocator ioAllocator;
   ProxyAllocator ioBlockAllocator;
 
-private:
-  // prevent unauthorized copies (Not implemented)
-  Thread(const Thread &);
-  Thread &operator=(const Thread &);
-
 public:
-  ink_thread start(const char *name, size_t stacksize, ThreadFunction f, void *a, void *stack);
+  ink_thread start(const char *name, void *stack, size_t stacksize, ThreadFunction const &f = ThreadFunction());
```

Comment:
I can't find a place where this the `ThreadFunction` is actually used. Unless you are planning to start using it in a subsequent patch, let's remove it?